tests/bsim/bluetooth/audio/src/bap_broadcast_source_test.c

```diff
@@ -37,6 +42,121 @@ static struct bt_bap_lc3_preset preset_16_1_1 = BT_BAP_LC3_BROADCAST_PRESET_16_1
 static K_SEM_DEFINE(sem_started, 0U, ARRAY_SIZE(broadcast_source_streams));
 static K_SEM_DEFINE(sem_stopped, 0U, ARRAY_SIZE(broadcast_source_streams));
 
+static uint8_t count_bits(enum bt_audio_location chan_allocation)
+{
+	uint8_t cnt = 0U;
+
+	while (chan_allocation != 0) {
+		cnt += chan_allocation & 1U;
+		chan_allocation >>= 1;
+	}
+
+	return cnt;
+}
+
+static void validate_stream_codec_cfg(const struct bt_bap_stream *stream)
```

Comment:
This should arguably have been a unit test, rather than a BSIM test, but if it's too much effort to move, it is OK to keep

Reply:
updated the existing unit test